packages/strapi-generate-plugin/files/admin/src/translations/index.js

```diff
@@ -0,0 +1,35 @@
+import ar from './ar.json';
```

Comment:
Can't we just put the en file and add a comment to explain how to add the other translations ?

Reply:
Since we'll drop all the translations files I thought it was faster to import them all.